packages/typespec-azure-core/lib/models.tsp

```diff
@@ -75,6 +75,7 @@ model SkipQueryParameter {
 @doc("Provides the standard 'maxpagesize' query parameter for list operations.")
 model MaxPageSizeQueryParameter {
   @query
```

Comment:
`top` and `skip` properties above should also the corresponding decorator `@pageSize` and `@offset`